### PR TITLE
Ignore asset hashing for assets/files

### PR DESCRIPTION
**Pull request by radeksimko**
We need a static URL (which doesn't change with every website release) for a logo, so we can link it reliably from our Terraform Readme-s.

The Github raw URLs don't seem to be stable for some reason and images keep disappearing from Readme-s.

This change should not affect any website as we only store `press-kit.zip` in such location which isn't hashed anyway (b/c it has zip extension):

https://github.com/hashicorp/consul/tree/master/website/source/assets/files
https://github.com/hashicorp/terraform-website/tree/master/content/source/assets/files
https://github.com/hashicorp/nomad/tree/master/website/source/assets/files
https://github.com/hashicorp/vault/tree/master/website/source/assets/files
https://github.com/mitchellh/vagrant/tree/master/website/source/assets/files
https://github.com/hashicorp/packer/tree/master/website/source/assets/files

TL;DR We need `https://www.terraform.io/assets/files/terraform-logo.svg` , _not_ `https://www.terraform.io/assets/files/terraform-logo-3f10732f.svg`

### CODE REVIEW

**Comment by sethvargo** [comment]
Hi @radeksimko 

Thank you for the PR.

Can you point to areas where the raw git URLs are failing? If you move the file in source, the raw git will fail, but that's the same behavior as if it's hosted on a URL.

I'm concerned about this PR because we'd have two logo assets to manage. This has been a huge hassle in the past - we had multiple logos scattered across all the projects. When we rebrand or make a small change, we have to remember to update 5000 logos. In the refactor, we got everything down to a single svg logo that is manipulated via CSS when changes are required. This would be introducing another asset to manage and understand the rollout effect for.

https://github.com/hashicorp/terraform/pull/15633 proves my point even further. Now the website team has to be aware of the required file structure instead of each team owning that themselves. You're effectively pushing the burden onto the release engineering team to manage this file, when I believe it should be the project teams' responsibilities.

I see exactly two issues on Terraform related to this:

- https://github.com/hashicorp/terraform/pull/13507
- https://github.com/hashicorp/terraform/pull/15633

Both of them are PRs (not issues), and I fixed one of them.

TL;DR we shouldn't be linking logos from the website onto the README when the file is sitting in that same repo. If a team changes their website layout, they should be responsible for updating the links in their README.

**Comment by sethvargo** [comment]
I did some digging. https://github.com/hashicorp/terraform-website/commit/ecad4213131280c157e894fc9b4f8c5c89726f06 and some prior commits moved things from `source/*` to `content/source/*` in the repo layout. This broke the logos because that _changes_ the raw git URL:

Here's an example from the DO provider:

- (current value, broken) https://cdn.rawgit.com/hashicorp/terraform-website/master/source/assets/images/logo-text.svg

- (new value, works) https://cdn.rawgit.com/hashicorp/terraform-website/master/content/source/assets/images/logo-text.svg

When you move files, it affects the layouts. 

There were also grumblings of the logos failing to render after they are added to so many repos. If you use rawgit.com, you'll get rate limited. This is why we use cdn.rawgit.com, which explicitly says:

> No traffic limits or throttling. Files are served via StackPath's super fast global CDN.

You could also lock to a specific commit hash if you wanted to avoid this in the future. However, if we ever update the logo, your team would be responsible for updating the link everywhere:

> Use a specific tag or commit hash in the URL (not a branch). Files are cached permanently based on the URL. Query strings are ignored.

- (cached and locked) https://raw.githubusercontent.com/hashicorp/terraform-website/ad2e24ced856a19c672360c02ae1eb61bd72e467/content/source/assets/images/logo-text.svg

Even if the repo layout changes, that URL will work. However, if the design team tweaks the logo, you'll need to go through and update the commit hash for all the values.

A third option is to not include the logo in the providers and just link back to terraform.io.

**Comment by radeksimko** [comment]
@sethvargo Thanks for the detailed explanation - it makes sense and I'm ok with closing this.

Also thanks @grubernaut for solving the problem the way you proposed. 🎉 👍
